src/SFA.DAS.Commitments.Database/SFA.DAS.Commitments.Database.sqlproj

```diff
@@ -110,6 +110,8 @@
     <Build Include="StoredProcedures\GetApprenticeshipUpdatesByDateAndStatus.sql" />
     <Build Include="StoredProcedures\GetActiveApprenticeships.sql" />
     <Build Include="StoredProcedures\SetTransferApproval.sql" />
+    <Build Include="Tables\JobProgress.sql" />
+    <Build Include="Tables\IntegrationTestIds.sql" />
   </ItemGroup>
```

Comment:
Does this mean we will place these tables in our now test DBs

Reply:
The JobProgress table is a new table that was created for an earlier EPA story that hasn't been merged yet. It's not integration test specific, although we do add an integration test specific column to store the schema version of the subset of the database the integration tests rely on.

The IntegrationTestIds will currently end up in the production database, which is far from ideal. I did consider creating it through a script in the integration test fixture setup, but didn't due to time constraints.
We could potentially run the integration tests against live (with further work on the tests) if we fixed the slowdown issues (in the spirit of Chaos Engineering which tests production!).